docs/deploy-configure.md

```diff
@@ -84,3 +122,11 @@ To upgrade this resource from `4.0.6` to `4.2.7`:
    ```
    kubectl apply -f <example>.yaml --namespace <my-namespace>
    ```
+
+## Deploy Replica Sets on OpenShift
+
+To deploy the operator on OpenShift you will have to provide the environment variable `MANAGED_SECURITY_CONTEXT` set to `true` for both the mongodb and mongodb agent containers, as well as the operator deployment.
```

Comment:
Again, I'm not sure if were talking about containers named "mongodb" or MongoDB instances here.

Reply:
updated. asked SME for guidance above just in case.

Reply:
in this case we are talking about 2 containers, one with the name of "mongodb" and one called "mongodb-agent". Not about a mongod process.

Reply:
@chatton updated. made it clearer that these were container names.

Reply:
@jwilliams-mongo formatting `mongodb` and `mongodb agent` might help make it clear that those are container names.

Reply:
Done. See https://github.com/mongodb/mongodb-kubernetes-operator/pull/247/files#diff-f78009da6728161bca1880b44e8c39f4d02497f945c161e525ae024a68935c11R128